src/binary/runBinary.ts

```diff
@@ -13,14 +14,17 @@ export default async function runBinary(
 
   const context = getTabnineExtensionContext();
   const args: string[] = [
-    "--client=vscode",
+    ON_PREM ? "--client=vscode_on_prem" : "--client=vscode",
     "--no-lsp=true",
     tabnineExtensionProperties.logFilePath
       ? `--log-file-path=${tabnineExtensionProperties.logFilePath}`
       : null,
+    ON_PREM ? "--onprem=true" : null,
+    ON_PREM ? "--no_bootstrap" : null,
```

Comment:
No need to pass it, it won't bootstrap anyway

Reply:
I'm not sure about that, depends on the build